src/commands/convert.ts

```diff
@@ -46,6 +46,11 @@ export default class Convert extends Command {
           this.log(`URL ${specFile.getFileURL()} successfully converted!`);
         }
       }
+      
+      if (typeof convertedFile === 'object') {
+        convertedFile = JSON.stringify(convertedFile);
```

Comment:
```suggestion
        convertedFile = JSON.stringify(convertedFile, null, 4);
```

more details in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

this will make your stringified JSON formatted nice, so humans can read it. Test it first, I didn't 😄